src/clr-angular/utils/i18n/common-strings.default.ts

```diff
@@ -62,4 +62,6 @@ export const commonStringsDefault: ClrCommonStrings = {
   daypickerSRCurrentDecadePhrase: 'The current decade is {DECADE_RANGE}',
   // Stack View
   stackViewChanged: 'Value changed.',
+  // Vertical Nav
+  verticalNavToggle: 'Toggle Vertical Nav',
```

Comment:
We don't want to say this, it wouldn't necessarily make sense. It should probably just be `Toggle navigation group`.

Reply:
I'm currently experiencing some issues with the environment, again. I'll update this PR as soon as I solve these issues.